test/externalTests/colony.sh

```diff
@@ -31,18 +31,17 @@ function test_fn { yarn run test:contracts; }
 function colony_test
 {
     OPTIMIZER_LEVEL=3
-    FORCE_ABIv2=false
     CONFIG="truffle.js"
 
-    truffle_setup https://github.com/solidity-external-tests/colonyNetwork.git develop_070
-    run_install install_fn
+    truffle_setup "$SOLJSON" https://github.com/solidity-external-tests/colonyNetwork.git develop_070
```

Comment:
This looks like a non shellcheck related change, does it belong in this PR?

Reply:
Shellcheck complains about `SOLJSON` being potentially uninitialized - because it's supposed to be set by the script and then used by `common.sh`.

So it's related, it's just not the most minimal fix I could do :) I could have just added some dummy initialization at the top of the script but these globals make the external test scripts hard to follow for me so I took the occasion to eliminate some of them.

Reply:
Does shellcheck complain about https://github.com/koalaman/shellcheck/wiki/SC2154?

Reply:
Yes. Exactly.

Reply:
Ah, wait. I think that was the error for `FORCE_ABIv2`. For this one it was that `soljson` command exists and `SOLJSON` looks like a wrongly capitalized version of it or something like that. That's why it only complained about these two and was fine with `DIR` and the other fully capitalized globals.